abs.yaml

```diff
@@ -0,0 +1,8 @@
+build_env_vars:
+  ANACONDA_ROCKET_ENABLE_PY313 : yes
+
+channels:
+  - https://staging.continuum.io/prefect/fs/pyinstaller-hooks-contrib-feedstock/pr4/726bfbb
+
+content_signing:
+  win-64: False
```

Comment:
The issue with signing on Windows seems to be solved.

Reply:
I will add a comment when I remove the staging channel. And for visibility, the artifacts are still being signed in the logs:

`Staging artifacts to https://staging.continuum.io/prefect/fs/pyinstaller-feedstock/pr11/950dee7...

staging_upload
Digitally signing each artifact...`

Reply:
Thanks!